tests/lib/rules/no-spaced-func.js

```diff
@@ -33,7 +33,8 @@ ruleTester.run("no-spaced-func", rule, {
         "( f()() )(0)",
         "(function(){ if (foo) { bar(); } }());",
         "f(0, (1))",
-        "describe/*.only*/('foo', function () {});"
+        "describe/*.only*/('foo', function () {});",
```

Comment:
Can you remove `.only` please? Even though it's commented out, it still scary:-)